massa-ledger-worker/src/ledger_db.rs

```diff
@@ -31,16 +31,16 @@ use std::{
 #[cfg(feature = "testing")]
 use massa_models::amount::{Amount, AmountDeserializer};
 
-const LEDGER_CF: &str = "ledger";
-const METADATA_CF: &str = "metadata";
-const OPEN_ERROR: &str = "critical: rocksdb open operation failed";
-const CRUD_ERROR: &str = "critical: rocksdb crud operation failed";
-const CF_ERROR: &str = "critical: rocksdb column family operation failed";
-const LEDGER_HASH_ERROR: &str = "critical: saved ledger hash is corrupted";
-const KEY_LEN_SER_ERROR: &str = "critical: key length serialization failed";
-const SLOT_KEY: &[u8; 1] = b"s";
-const LEDGER_HASH_KEY: &[u8; 1] = b"h";
-const LEDGER_HASH_INITIAL_BYTES: &[u8; 32] = &[0; HASH_SIZE_BYTES];
+pub const LEDGER_CF: &str = "ledger";
+pub const METADATA_CF: &str = "metadata";
+pub const OPEN_ERROR: &str = "critical: rocksdb open operation failed";
+pub const CRUD_ERROR: &str = "critical: rocksdb crud operation failed";
+pub const CF_ERROR: &str = "critical: rocksdb column family operation failed";
+pub const LEDGER_HASH_ERROR: &str = "critical: saved ledger hash is corrupted";
+pub const KEY_LEN_SER_ERROR: &str = "critical: key length serialization failed";
+pub const SLOT_KEY: &[u8; 1] = b"s";
+pub const LEDGER_HASH_KEY: &[u8; 1] = b"h";
+pub const LEDGER_HASH_INITIAL_BYTES: &[u8; 32] = &[0; HASH_SIZE_BYTES];
```

Comment:
remove pub here please

Reply:
Corrected. Apologies, when reverting the file-split, i removed the `(crate)`, but for some reason didn't remove the `pub` also.

Thanks for the catch.